TypeCobol.Test/Parser/Programs/Cobol85/ProgramEndMismatchNested0.rdz.cbl

```diff
@@ -0,0 +1,19 @@
+﻿       IDENTIFICATION DIVISION.
+       PROGRAM-ID. Main.
+      
+       IDENTIFICATION DIVISION.
+       PROGRAM-ID. Nested0.
+      
+       END PROGRAM Toto.
+      
+       IDENTIFICATION DIVISION.
+       PROGRAM-ID. Nested1.
+      
+       END PROGRAM Nested1.
+      
+       END PROGRAM Main.
```

Comment:
I don't think this node is an orphan as 
`END PROGRAM Toto` closes `Nested0` because no program in the stack can match `Toto`, 
`Nested1` is closed regularly,
`END PROGRAM Main` closes `Main`.

Reply:
Yes indeed.